examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -1,37 +1,56 @@
-# Basic Python CI With Cloud Source Repositories and Cloud Build
+# Basic Python Continuous Integration (CI) With Cloud Source Repositories (CSR) and Cloud Build
 
 ## Overview
-This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
-
-The basic steps we need to follow are:
-1. Create a new project in Cloud Source Repository and clone it to your machine
-2. Create your code and unit tests
-3. Create a Python Cloud Builder (more below)
-4. Create a cloud build trigger (more below)
-5. Create a cloudbuild.yaml file that executes your tests using your python cloud builder.
-
-## 1. Create a new project in Cloud Source Repository
-* Go to https://source.cloud.google.com/
-* Click 'Add repository'
-* Choose 'Create new repository'
-* Specify a name, and your project name
-* Follow the instructions to 'git clone' the empty repo to your workstation and use it like you would a git repo
+This repo contains example code and instructions that show you how to use CSR and 
+Cloud Build to automatically run unit tests and pylint upon code check-in. By following this tutorial you will learn
+how to build basic Python continuous integration pipelines on Google Cloud Platform (GCP).
+
+By following along with this example you will learn how to:
+1. Create a new project in CSR and clone it to your machine.
+2. Create your code and unit tests.
+3. Create a custom container called a cloud builder that Cloud Build will use to run your Python tests.
+4. Create a cloud build trigger that will tell Cloud Build when to run.
+5. Tie it all together by creating a cloudbuild.yaml file that tells Cloud Build how to execute your tests
+ when the cloud build trigger fires, using the custom cloud builder you created.
+ 
+In order to follow along with this tutorial you'll need to:
+* Create or have access to an existing [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+* Install and configure the [Google Cloud SDK](https://cloud.google.com/sdk).
+
+## 1. Create a new project in Cloud Source Repositories
+You'll start by creating a new repository in CSR, copying the files in this example into the CSR repository, and 
+commiting them to your new repository.
+
+* Go to https://source.cloud.google.com/.
+* Click 'Add repository.'
+* Choose 'Create new repository.'
+* Specify a name, and your project name.
+* Follow the instructions to 'git clone' the empty repo to your workstation.
+* Copy the files from this example into the new repo.
+* Add the files to the new repo with 'git add .'
+* Commit and push these files in the new repo with the commands 'git commit -m 'inital commit' and 'git push origin master.'
```

Comment:
When you're giving the user a command to run, but it in markdown as a code example. You do that with three backticks. 
\```code to run or maybe a filename or whatever```
```code to run or maybe a filename or whatever```

you also get basic syntax highlighting if you use 3 backticks then the name of a language.
\```python 
print ('yes')
\```
```python 
print ('yes')
```

\```bash
echo $PROJECT
\```
```bash
echo $PROJECT
```

Reply:
Also, check go/cl-descriptions for information on commit message style, "inital commit" doesn't meet them. The basics are imperative style and make it a complete sentence, e.g., "Copy Cloud Build Python template to empty repo to start project.", though I'll let "Initial Commit." slide :).

Reply:
Ugh my apologies, you clearly know about the backtick stuff, I'm so sorry for the obnoxious assumption. But you should use it here.

Reply:
Done!